lighthouse-viewer/app/src/fileuploader.js

```diff
@@ -1,113 +0,0 @@
-/**
- * @license Copyright 2016 Google Inc. All Rights Reserved.
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
- */
-'use strict';
-
-/* global logger */
-
-/**
- * Manages drag and drop file input for the page.
- * @class
- */
-class FileUploader {
-  /**
-   * @param {function()} fileHandlerCallback Invoked when the user chooses a new file.
-   */
-  constructor(fileHandlerCallback) {
-    this.dropZone = document.querySelector('.drop_zone');
-    this.placeholder = document.querySelector('.viewer-placeholder');
-    this._fileHandlerCallback = fileHandlerCallback;
-    this._dragging = false;
-
-    this.addHiddenFileInput();
-    this.addListeners();
-  }
-
-  addHiddenFileInput() {
-    this.fileInput = document.createElement('input');
-    this.fileInput.id = 'hidden-file-input';
-    this.fileInput.type = 'file';
-    this.fileInput.hidden = true;
-    this.fileInput.accept = 'application/json';
-
-    this.fileInput.addEventListener('change', e => {
```

Comment:
> We don't need the change handler anymore? 

this is in LighthouseReportViewer now (Github helpfully collapses the diff of that file for some reason)